### ISSUE TITLE
Remove StyledComponents to simplify development, improve a11y

### ISSUE THREAD

**Issue by marcysutton** [charliewilco/a11y-camp#7]
Now that I'm working more on this site and building out the missing content pages, I'm finding some things that should be simplified to make the site easier to develop and help with accessibility. In particular, the use of StyledComponents adds some extra layers of complexity:

- A lot of inline styles that are not portable in between files
- Using CSS-in-JS raises the barrier to entry for HTML and CSS developers who may contribute in the future
- Cryptic, cached CSS class names that are very difficult to override with [user stylesheets](https://www.viget.com/articles/inline-styles-user-style-sheets-and-accessibility/)

Different approaches would be to use CSS modules, plain CSS imports per page or component, or a global CSS file. I'd be open to trying CSS modules since they are recommended by Gatsby, but we'll have to test to see whether the cached class names thing still causes problems (e.g. are they modified with every build or do they stay stable)? https://www.gatsbyjs.org/docs/css-modules/

**Comment by marcysutton** [comment]
Here's some info from Dustin on the Gatsby core team with regard to cached class names!

>The first portion with CSS Modules remains constant (based on file system path) but the hash probably changes based on content changes.

>If the goal is to provide something overwritable maybe just provide a second class name?

```
div className={`{styles.whatever} something-easily-themed`}
```
